src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/InputHideUtilityClassConstructorIgnoreAnnotationByFullyQualifiedName.java

```diff
@@ -0,0 +1,19 @@
+/*
+HideUtilityClassConstructor
+ignoreAnnotatedBy = java.lang.Deprecated
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.design.hideutilityclassconstructor;
+
+@Deprecated // violation
```

Comment:
please add violation messages to all comments "// violation"
same in Examples.

Reply:
The [violation message](https://github.com/checkstyle/checkstyle/blob/a4b0fc7b36dd1326a6c37d6b2b10070974967448/src/main/resources/com/puppycrawl/tools/checkstyle/checks/design/messages.properties#L7) is quite long, and checkstyle is failing on LineLenght, so I did not include it. What is a nice way to overcome this issue? Maybe add the message in a separate line?

Reply:
Message can be partial, usage of any meaningful part of it is enough

Reply:
https://github.com/checkstyle/checkstyle/blob/a4b0fc7b36dd1326a6c37d6b2b10070974967448/src/main/resources/com/puppycrawl/tools/checkstyle/checks/design/messages.properties#L7

`should not have a public or default constructor` should be good

Reply:
Done